include/fmt/format.h

```diff
@@ -1766,12 +1773,13 @@ inline auto write_significand(OutputIt out, T significand, int significand_size,
 }
 
 template <typename OutputIt, typename DecimalFP, typename Char>
-auto write_float(OutputIt out, const DecimalFP& fp,
-                 const basic_format_specs<Char>& specs, float_specs fspecs,
-                 locale_ref loc) -> OutputIt {
+FMT_CONSTEXPR20 auto write_float(OutputIt out, const DecimalFP& fp,
+                                 const basic_format_specs<Char>& specs,
+                                 float_specs fspecs, locale_ref loc)
+    -> OutputIt {
   auto significand = fp.significand;
   int significand_size = get_significand_size(fp);
-  static const Char zero = static_cast<Char>('0');
+  constexpr const Char zero = static_cast<Char>('0');
```

Comment:
Why change this?

Reply:
There is no way to have the `static` variable in the function marked as `constexpr`, and `write_float` is marked so, with C++20. Of course, it could result in worse codegen, but I didn't properly check that.